undertow/src/main/java/org/wildfly/extension/undertow/ActiveRequestTrackerDefinition.java

```diff
@@ -0,0 +1,112 @@
+package org.wildfly.extension.undertow;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import io.undertow.server.RequestStatistics;
+import io.undertow.server.handlers.ActiveRequestTrackerHandler;
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.OperationStepHandler;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PersistentResourceDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.SimpleOperationDefinition;
+import org.jboss.as.controller.SimpleOperationDefinitionBuilder;
+import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.capability.DynamicNameMappers;
+import org.jboss.as.controller.capability.RuntimeCapability;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.jboss.msc.service.ServiceController;
+import org.jboss.msc.service.ServiceName;
+import org.jboss.msc.service.ServiceRegistry;
+
+public class ActiveRequestTrackerDefinition extends PersistentResourceDefinition {
```

Comment:
I am generally skeptical of single attribute child resources.  Is this child resource really warranted?  or might it be better represented as an attribute of its parent resource?

Reply:
After looking at this change in more detail, it seems that this feature has 2 bits of configuration, a boolean flag on the host, and a single property child resource.
So, a user would either:
* Keep the flag disabled, and not specify an active-request-tracker resource
* Enable the flag, and specify an active-request-tracker resource

That would suggest that the boolean flag is superfluous, and the feature can enabled or disabled depending on whether the child resource is defined or not.

Secondly, is there any possibility that this resource would ever have additional attributes?  If not, it seems to me that this is better represented by a single "active-request-tracker" (or similar) attribute of type ModelType.String defined within the Host resource.  WDYT?